### PR TITLE
DDCE-5184 - Play 3.0 upgrade

### PR DESCRIPTION
**Pull request by frikit**

### CODE REVIEW

File: it/test/repositories/ActiveSessionRepositorySpec.scala [hunk 1]
```diff
@@ -23,14 +23,15 @@ import org.scalatest.matchers.must.Matchers
 import org.scalatest.wordspec.AnyWordSpec
 import org.scalatest.{BeforeAndAfterEach, OptionValues}
 import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import repositories.ActiveSessionRepositoryImpl
```
Comment (MRBeresford) [inline it/test/repositories/ActiveSessionRepositorySpec.scala:26]: needed?

**Review by MRBeresford — COMMENTED** [review]
Looks good just two questions about imports I dont think you need (but I may be wrong)

File: it/test/repositories/PlaybackRepositorySpec.scala [hunk 2]
```diff
@@ -26,6 +25,7 @@ import org.scalatest.wordspec.AnyWordSpec
 import org.scalatest.{BeforeAndAfterEach, OptionValues}
 import play.api.libs.json.Json
 import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import repositories.PlaybackRepositoryImpl
```
[the remark this answers is not in the document]
Comment (frikit) [reply it/test/repositories/PlaybackRepositorySpec.scala:28]: fixed

File: it/test/repositories/ActiveSessionRepositorySpec.scala (hunk 1 above)
Comment (frikit) [reply it/test/repositories/ActiveSessionRepositorySpec.scala:26]: fixed

**Review by MRBeresford — APPROVED** [review]
LGTM

**Pull request merged**